### PR TITLE
swatch-2690: event schema updates

### PR DESCRIPTION
**Pull request by kflahert**
Jira issue: SWATCH-2690

## Description

Improve event schema descriptions and examples for use by other teams

### CODE REVIEW

File: swatch-model-events/schemas/event.yaml [hunk 1]
```diff
@@ -94,6 +124,11 @@ properties:
           description: Metric ID for the measurement. (e.g. Cores, Sockets)
```
Comment (barnabycourt) [inline swatch-model-events/schemas/event.yaml:124]: Should there be information about what this is being matched up to in swatch product-config?

Comment (kflahert) [reply swatch-model-events/schemas/event.yaml:124]: Yeah I'll elaborate on that more, thanks!

File: swatch-model-events/schemas/event.yaml [hunk 2]
```diff
@@ -5,54 +5,84 @@ properties:
...
+      - 2024-08-29T12:00:00Z
   record_date:
     description: The date that the EventRecord was persisted (NOTE Automatically set when persisted)
     type: string
     format: date-time
     required: false
+    examples:
+      - 2024-08-29T12:00:00Z
   expiration:
     description: End of the range for the event (exclusive).
```
Comment (mstead) [inline swatch-model-events/schemas/event.yaml:71]: Note that this is not currently being used.

File: swatch-model-events/schemas/event.yaml [hunk 3]
```diff
@@ -171,31 +214,43 @@ properties:
...
-    description: The azure subscription ID used for billing
+    description: The azure subscription ID used for billing. Required for PAYG azure instances.
     type: string
     existingJavaType: java.util.Optional<String>
     required: false
+    examples:
+      - f49a1a52-7c09-4c2f-a227-c8680c7b5a17
   product_tag:
-    description: Product tag
+    description: Product tag of the associated offering.
```
Comment (mstead) [inline swatch-model-events/schemas/event.yaml:247]: Could use a little more context around this one. What should this match in the config?

**Review by mstead — APPROVED** [review]

**Pull request merged**